src/linpred.jl

```diff
@@ -138,12 +138,12 @@ function delbeta!(p::DensePredChol{T,<:CholeskyPivoted}, r::Vector{T}) where T<:
     if rnk == length(delbeta)
         ldiv!(ch, delbeta)
     else
-        permute!(delbeta, ch.piv)
+        permute!(delbeta, ch.p)
```

Comment:
I couldn't find documentation of the `p` property. Am I just being dense or is it undocumented?

Reply:
It's new. ;-) https://github.com/JuliaLang/julia/pull/39964